From 375871f8df3c63f973f1d9f21c55f0618e87b682 Mon Sep 17 00:00:00 2001
From: Tomasz Figa <t.figa@samsung.com>
Date: Fri, 1 Mar 2013 18:15:48 +0100
Subject: [PATCH 0107/1302] ARM: dts: exynos4210-trats: Add cpufreq node

This patch adds cpufreq node with board-specific parameters to enable
CPU frequency scaling on Trats board.

Signed-off-by: Tomasz Figa <t.figa@samsung.com>
Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>
---
 arch/arm/boot/dts/exynos4210-trats.dts | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm/boot/dts/exynos4210-trats.dts b/arch/arm/boot/dts/exynos4210-trats.dts
index 444474b..6f444d2 100644
--- a/arch/arm/boot/dts/exynos4210-trats.dts
+++ b/arch/arm/boot/dts/exynos4210-trats.dts
@@ -432,4 +432,10 @@
 			memory-region = <&mfc_r_mem>;
 		};
 	};
+
+	cpufreq {
+		/* No freq_table property to use default table. */
+		vdd_arm-supply = <&varm_breg>;
+		status = "okay";
+	};
 };
-- 
1.8.3.2

